tests/resources/test-types.json

```diff
@@ -1608,7 +1608,7 @@
             "linkedTestCode": null
           }
         ],
-        "testTypeClassification": "Annual NO CERTIFICATE",
+        "testTypeClassification": "Annual With Certificate",
```

Comment:
enabling generation for adr test types is part of 3952, which I think you got by rebasing. But I think it can go with your ticket cuz it won't impact anything